docs/src/app_dev/daml_api/token_standard.rst

```diff
@@ -0,0 +1,291 @@
+
+To execute a choice on a contract implementing a Token Standard interface for external parties,
+you must call the `prepare <https://github.com/hyperledger-labs/splice/blob/902f7069602b7bbce49293cabe7f162cfd8dbffb/canton/community/ledger/ledger-json-api/src/test/resources/json-api-docs/openapi.yaml#L1553>`_
+and `execute <https://github.com/hyperledger-labs/splice/blob/902f7069602b7bbce49293cabe7f162cfd8dbffb/canton/community/ledger/ledger-json-api/src/test/resources/json-api-docs/openapi.yaml#L1585>`_
+endpoints of the participant.
+For non-external parties, you can just use the `submit-and-wait endpoint <https://github.com/hyperledger-labs/splice/blob/902f7069602b7bbce49293cabe7f162cfd8dbffb/canton/community/ledger/ledger-json-api/src/test/resources/json-api-docs/openapi.yaml#L6>`_.
+
+In both cases, you must include an ``ExerciseCommand`` in your payload with the following fields:
+
+* ``templateId``: the interface id of the contract you want to exercise the choice on. For example, ``#splice-api-token-transfer-instruction-v1:Splice.Api.Token.TransferInstructionV1:TransferInstruction``.
+* ``contractId``: the contract id of the contract you want to exercise the choice on. Typically, you'll get this from :ref:`the current ACS of a party <token_standard_usage_reading_contracts>`.
+* ``choice``: the name of the choice you want to execute. For example, ``TransferInstruction_Accept``.
+* ``choiceArgument``: the arguments that will be passed to the Daml choice. These will be decoded from JSON.
+
+Where a ``context`` is required as part of the ``choiceArgument``, it can be fetched from the corresponding registry:
+
+* `To accept a TransferInstruction </app_dev/token_standard_openapi/index.html#post--registry-transfer-instruction-v1-transferInstructionId-choice-contexts-accept>`_
+* `To reject a TransferInstruction </app_dev/token_standard_openapi/index.html#post--registry-transfer-instruction-v1-transferInstructionId-choice-contexts-reject>`_
+* `To withdraw a TransferInstruction </app_dev/token_standard_openapi/index.html#post--registry-transfer-instruction-v1-transferInstructionId-choice-contexts-withdraw>`_
+* `To withdraw an Allocation </app_dev/token_standard_openapi/index.html#post--registry-allocations-v1-allocationId-choice-contexts-withdraw>`_
+* `To cancel an Allocation </app_dev/token_standard_openapi/index.html#post--registry-allocations-v1-allocationId-choice-contexts-cancel>`_
+
+The response of these endpoints include two fields:
+
+* ``choiceContextData``: to be passed as ``context`` in the ``choiceArgument``.
+* ``disclosedContracts``: to be passed in the submit or prepare request.
+
+.. warning::
+
+  Note that ``AllocationRequest_Reject`` and ``AllocationRequest_Withdraw`` should be called with an empty choice context.
+  This is currently there as a potential future extension.
+
+
+
+.. _token_standard_usage_workflows:
+
+Token Standard Workflows
+------------------------
```

Comment:
The sections below lack context as to why these are relevant and what's their difference.

I'd suggest that you reference the corresponding sections in the standard:
- https://github.com/global-synchronizer-foundation/cips/blob/main/cip-0056/cip-0056.md#direct-peer-to-peer--free-of-payment-fop-transfer-workflow
- https://github.com/global-synchronizer-foundation/cips/blob/main/cip-0056/cip-0056.md#delivery-versus-payment-dvp-transfer-workflows

And inline a copy of them interleaved with the technical content that you reference below.

I'd also consider moving this section up towards the top, as it gives context for the different kinds ledger interactions explained in the above sections.

wdyt?